### ISSUE TITLE
1D Gaussian with unknown mean and variance

### ISSUE THREAD

**Issue by ErikOsinga** [ErikOsinga/master_research_project#4]
Probably better to test the networks capabilities on 2 parameters with a simpler example. 
Thus will define theta = [mu,sigma] and see if the network works on 1D Gaussian data

**Comment by ErikOsinga** [comment]
Seems the main problem was not using the ADAM optimizer. At the moment it is working quite well, if the learning rate is not too high. Now testing various different parameters.

**Comment by ErikOsinga** [comment]
After implementing 2D posterior plotting, I think we can close this issue. The latest result for this 1D Gaussian with unknown mean and variance is now:

![variables_vs_epochs_1013](https://user-images.githubusercontent.com/12582441/50692147-38924f80-1033-11e9-9090-eca1cec42675.png)

Which results in the following approximate 2D posterior, the blue lines mark the true values. 

![pmc_abc_1013_2d](https://user-images.githubusercontent.com/12582441/50692282-a76fa880-1033-11e9-82f1-e5bdbe958541.png)


This was achieved with the following modelsettings:

> Version,Learning rate,Keep rate,num_epochs,n_train,delta_theta,number of simulations,fiducial θ,differentiation fraction,input shape,number of summaries,calculate MLE,prebuild,save file,wv,bb,activation,α,hidden layers,Final detF train,Final detF test
> 
> 1013,1e-05,0.6,10000,1,[0.1 0.1],10000,[0. 1.],0.05,[10],2,True,True,Models/data/model1013,0.0,0.1,leaky_relu,0.01,"[256, 256, 256]",47.37,42.19
>

**Comment by mjvakili** [comment]
Excellent! closing this issue
